autosklearn/pipeline/components/data_preprocessing/feature_reduction/truncated_svd.py

```diff
@@ -0,0 +1,77 @@
+from typing import Dict, Optional, Tuple, Union
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+import ConfigSpace.hyperparameters as CSH
+
+import numpy as np
+
+from autosklearn.pipeline.base import DATASET_PROPERTIES_TYPE, PIPELINE_DATA_DTYPE
+from autosklearn.pipeline.components.base import AutoSklearnPreprocessingAlgorithm
+from autosklearn.pipeline.constants import DENSE, SPARSE, UNSIGNED_DATA, INPUT
+
+from sklearn.decomposition import TruncatedSVD
+
+
+class FeatureReduction(AutoSklearnPreprocessingAlgorithm):
```

Comment:
https://github.com/automl/auto-sklearn/pull/1276#discussion_r742060158 is not addressed yet.